pvlib/tests/test_irradiance.py

```diff
@@ -15,7 +15,8 @@
     assert_frame_equal,
     assert_series_equal,
     requires_ephem,
-    requires_numba
+    requires_numba,
+    fail_on_pvlib_version
```

Comment:
```suggestion
    fail_on_pvlib_version,
```
Adding a comma avoids the line showing up in the diff when adding a new entry to the list (see for example how the requires_numba is showing up).